docs/resources/data-management.md

```diff
@@ -0,0 +1,27 @@
+# Data management
+
+A challenging aspect of tutorial development is how to manage datasets. Ideally, the data you use will be publicly accessible and permanent for reproducibility of your computational examples.
+
+```{important}
+Remember, a hackweek tutorial is *learning-oriented* and should guide participants through a step-wise process with a meaningful outcome.
+```
+
+## Make your tutorial data publicly accessible
+*We recommend using small (<100 MB) example datasets for your tutorial*. If you typically work with large datasets, consider working with a small subset to get your ideas across.
```

Comment:
Could we recommend even smaller datasets?
I would be bold and recommend around 10 MB. Main concern here ist that the repository with a lot of large sample datasets for each tutorial will get big quick. This is without thinking that a tutorial can potentially have multiple versions of a sample dataset

Reply:
I'll try to clarify that a dataset (e.g. CSV <10MB) is okay to add to a repository along with code, and >10MB should be stored elsewhere, such as a separate repository or via Zenodo